datahub-web-react/src/app/entity/dataset/shared/DatasetStatsSummary.tsx

```diff
@@ -40,6 +54,19 @@ export const DatasetStatsSummary = ({
             <StatText>
                 <TableOutlined style={{ marginRight: 8, color: ANTD_GRAY[7] }} />
                 <b>{formatNumberWithoutAbbreviation(rowCount)}</b> rows
+                {(!!columnCount && (
+                    <>
+                        , `<b>{formatNumberWithoutAbbreviation(rowCount)}</b> columns
+                    </>
+                )) ||
+                    undefined}
```

Comment:
nitty nit: the `|| undefined` here isn't necessary! if you do `{!!columnCount && (...` it will only render what comes next if `!!columnCount` is truthy

Reply:
awesome! will change